lib/puppet/type/posix_acl.rb

```diff
@@ -131,11 +131,9 @@ def retrieve
       provider.permission
     end
 
+    # Remove permission bits from an ACL line, eg:
+    # 'user:root:rwx' becomes 'user:root:'
     def strip_perms(pl)
-      desc = "Remove permission bits from an ACL line, eg:
```

Comment:
This isn't a puppet param/property, so I don't think there's a 'desc' DSL here.

Reply:
LGTM